devel/libblocksruntime/Portfile

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libblocksruntime
+version             0.4.1
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         gmail.org:adsun701
+description         compiler-rt Blocks runtime library for Clang
+long_description    libblocksruntime is a target-independent implementation of Apple "Blocks" runtime interfaces.
+homepage            https://compiler-rt.llvm.org
+master_sites        http://ftp.debian.org/debian/pool/main/libb/libblocksruntime/
```

Comment:
Is this really the best source for this, given that Debian isn't the LLVM project? Perhaps a fetch straight from the Subversion tag would be superior?

Reply:
LLVM does not provide libblocksruntime as a standalone source, but rather provides it as part of compiler-rt.

Reply:
Yes, sure, but you can certainly extract it directly with an svn command.

Reply:
The Debian source for libblocksruntime allows to install it as a standalone library using a configure script, without having to build the entire compiler-rt source.